AllASPFuncs.py

```diff
@@ -741,30 +741,31 @@ def get_object_info(ID, parq, band, snpath, roman_path, obj_type):
     return ra, dec, pointing, sca, start, end, peak
 
 
-
-def getWeights(cutout_wcs_list,size,snra,sndec, error = None, gaussian_std = 1000, cutoff = np.inf):
+def getWeights(cutout_wcs_list, size, snra, sndec, error=None,
+               gaussian_std=1000, cutoff=np.inf):
     wgt_matrix = []
     Lager.debug(f'Gaussian std in getWeights {gaussian_std}')
-    for i,wcs in enumerate(cutout_wcs_list):
-        xx, yy = np.meshgrid(np.arange(0,size,1), np.arange(0,size,1))
+    for i, wcs in enumerate(cutout_wcs_list):
+        xx, yy = np.meshgrid(np.arange(0, size, 1), np.arange(0, size, 1))
         xx = xx.flatten()
         yy = yy.flatten()
 
-        rara, decdec = wcs.toWorld(xx, yy, units = 'deg')
+        rara, decdec = wcs.toWorld(xx, yy, units='deg')
         dist = np.sqrt((rara - snra)**2 + (decdec - sndec)**2)
 
-        snx, sny = wcs.toImage(snra, sndec, units = 'deg')
+        snx, sny = wcs.toImage(snra, sndec, units='deg')
         dist = np.sqrt((xx - snx + 1)**2 + (yy - sny + 1)**2)
 
         wgt = np.ones(size**2)
-
-
         wgt = 5*np.exp(-dist**2/gaussian_std)
+
         wgt[np.where(dist > 4)] = 0
```

Comment:
Explain why the units of dist (wr.t. the bare value `4`) and why it's OK to throw out outliers and whether that affects the calculation.